test/gui/shared/scripts/bdd_hooks.py

```diff
@@ -64,7 +65,7 @@ def hook(context):
     if context.userData['clientSyncTimeout'] == '':
         context.userData['clientSyncTimeout'] = 60
     else:
-        context.userData['clientSyncTimeout'] = int(
+        context.userData['clientSyncTimeout'] = builtins.int(
```

Comment:
Squish's `int` will throw an error if `CLIENT_SYNC_TIMEOUT` is set in config.ini
```
No matching 'int(str)' overload found:
```